bats/fb-katello-container.bats

```diff
@@ -13,3 +13,13 @@ load fixtures/content
   CONTAINER_PULL_LABEL=$(echo "${ORGANIZATION_LABEL}-${PRODUCT_LABEL}-${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]')
   podman pull "${HOSTNAME}/${CONTAINER_PULL_LABEL}"
 }
+
+@test "push container to katello" {
+  tContainerPushSupported
+  tPackageExists podman || tPackageInstall podman
+  podman login "${HOSTNAME}" -u admin -p changeme
+  CONTAINER_PULL_LABEL=$(echo "${ORGANIZATION_LABEL}-${PRODUCT_LABEL}-${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]')
+  CONTAINER_PUSH_LABEL=$(echo "${ORGANIZATION_LABEL}/${PRODUCT_LABEL}/${CONTAINER_REPOSITORY_LABEL}-bats-$(date -u '+%s')"| tr '[:upper:]' '[:lower:]')
```

Comment:
Yes, you need to pull with `ORG-PRODUCT-REPO` but push with `ORG/PRODUCT/REPO`.

@ianballou this hurts :crying_cat_face:

Reply:
Aye it does - @ianballou is looking into whether we could switch to a `/` based naming scheme which is more common within the container world.

Reply:
interestingly, pushed stuff is then also only pull-able with `/` -- so `-` only applies to synced stuff? confusing!

Reply:
nah, let's keep as close to the default config as possible